modules/hcp-configuring-oauth.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-authentication-authorization.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-configuring-oauth_{context}"]
+= Configuring the internal OAuth server for a hosted cluster
+
+You can configure the internal OAuth server for your hosted cluster by using an OpenID Connect identity provider. Configuring the OAuth configuration avoids the `kubeadmin` user password from being exposed.
```

Comment:
> You can configure the internal OAuth server for your hosted cluster by using an OpenID Connect identity provider.

OpenID connect is not the only idp provider that we support. We support all identity providers for OpenShift (see siblings of https://docs.openshift.com/container-platform/4.15/authentication/identity_providers/configuring-htpasswd-identity-provider.html#identity-provider-overview_configuring-htpasswd-identity-provider) 

Replace: "Configuring the OAuth configuration avoids the `kubeadmin` user password from being exposed." with: "Adding any identity provider under the OAuth configuration removes the default `kubeadmin` user provider."

Reply:
Addressed

Reply:
- Added a redirecting link under additional resources to know more about supported identity providers.
- Replaced a sentence with ""Adding any identity provider under the OAuth config....."